cardea-server/src/main/java/ca/on/oicr/gsi/cardea/server/controller/CardeaApiController.java

```diff
@@ -41,6 +42,18 @@ public ZonedDateTime getDataTimestamp() {
     }
   }
 
+  @GetMapping("/djerba-cases/{requisitionName}")
+  public DjerbaCases getDjerbaCases(@PathVariable String requisitionName) {
+    if (requisitionName == null || requisitionName.isBlank()) {
+      throw new BadRequestException("must provide requisition name in URL");
+    }
```

Comment:
Is it possible to trigger this? `PathVariables` are required by default, so I think Spring will return an error before it gets here

Reply:
If `requisitionName` is a single space, it returns the error message on line 52 after it's failed to find any matching cases.